CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ Main (unreleased)
   - `diskstats_ignored_devices` is now `diskstats_device_exclude` in agent configuration.
   - `ignored_devices` is now `device_exclude` in flow configuration.
 
+- A number of components were updated now that type squashing is implemented in river. Please review the component docs.
+  - `prometheus.scrape` (@erikbaranowski)
```

Comment:
IMO, this should be changed to talk more about why we're doing it from the user's perspective: 

```suggestion
- Some blocks in Flow components have been merged with their parent block to make the block hierarchy smaller:
  - `prometheus.scrape > client > http_client_config` is merged into the `client` block. (@erikbaranowski)
```

We also need to write a section for this in the upgrade guide.